assets/src/ba_data/python/bastd/activity/coopscorescreen.py

```diff
@@ -1193,22 +1193,22 @@ def _show_world_rank(self, offs_x: float) -> None:
             Text(ba.Lstr(resource='coopSelectWindow.timeRemainingText'),
                  position=(-360, -70 - 100),
                  color=(1, 1, 1, 0.7),
-                 h_align='center',
-                 v_align='center',
-                 transition='fade_in',
+                 h_align=Text.HAlign.CENTER,
+                 v_align=Text.VAlign.CENTER,
+                 transition=Text.Transition.FADE_IN,
                  scale=0.8,
                  maxwidth=300,
                  transition_delay=2.0).autoretain()
-            self._tournament_time_remaining_text = Text('',
-                                                        position=(-360,
-                                                                  -110 - 100),
-                                                        color=(1, 1, 1, 0.7),
-                                                        h_align='center',
-                                                        v_align='center',
-                                                        transition='fade_in',
-                                                        scale=1.6,
-                                                        maxwidth=150,
-                                                        transition_delay=2.0)
+            self._tournament_time_remaining_text = Text(
+                '',
+                position=(-360, -110 - 100),
+                color=(1, 1, 1, 0.7),
+                h_align=Text.VAlign.CENTER,
```

Comment:
A good little example of the benefit of trying to get type-checking functioning before applying a fix: h_align was being given a Text.VAlign value here. (Though this didn't cause any problems in this particular case.)